### ISSUE TITLE
window.songs.get('content').pushObject(newSong);

### ISSUE THREAD

**Issue by tenzan** [balinterdi/rarwe-issues#108]
Hello!

Running `window.songs.get('content').pushObject(newSong);` in the console doesn't add the 4th song on the browser.

This is my commit https://github.com/tenzan/rarwe/commit/40e7b017446704aee5d828c81c8a90af7c97c577

Can you please look what I did wrong?

Thanks.

**Comment by ImmaculatePine** [comment]
+1. I have the same problem.

**Comment by balinterdi** [comment]
@tenzan I fixed it and sent you a pull request with the fix. The code has to change as folows:

```js
export default Ember.Controller.extend({
  songs: songs
});
```
```hbs
{{#each songs.sortedContent as |song|}}
    <li class="list-group-item">{{song.title}} by <em>{{song.band}}</em>
        <span class="rating pull-right">{{song.rating}}</span></li>
{{/each}}
```

So getting `sortedContent` on the songs collection needs to be moved to the template, from the controller. The list was not re-rendered in the template because the `songs: songs.get('sortedContent`)` is a one-time assignment, no observers are set up to act on change. In contrast, each-ing through an array in the template does set up that observer and that's why my solution works.

**Comment by balinterdi** [comment]
This is now fixed in the rock-and-roll-with-emberjs-20151009-2 release that just went out.

**Comment by tenzan** [comment]
Thanks a lot!
